be/src/vec/exec/vset_operation_node.cpp

```diff
@@ -174,6 +174,8 @@ Status VSetOperationNode<is_intersect>::init(const TPlanNode& tnode, RuntimeStat
         _child_expr_lists.push_back(ctxs);
     }
 
+    _probe_finished_children_index.assign(_child_expr_lists.size(), false);
```

Comment:
Any difference between do it in `init` and `alloc_resource ` ?

Reply:
`alloc_resource` will be called multi times(sink operator/source operator/probe sink operator)